code/_onclick/drag_drop.dm

```diff
@@ -1,19 +1,40 @@
+/atom/MouseDrag()
+	if(!dragging)
+		dragging = TRUE
+		drag_start = world.time
+
+/atom/proc/could_be_lag()
```

Comment:
Could you add a comment to this function explaining what its purpose is, and possibly change the function name to be a bit more descriptive about it dealing with click dragging?

Reply:
https://github.com/ParadiseSS13/Paradise/pull/21052/commits/d7b1c60523854173f455f567c3d40a4b44cf2902